scripts/test-telemetry-origin

```diff
@@ -19,7 +19,7 @@ def iter_data(path: Union[Path, str]) -> Generator[str, str, int]:
             continue
         filename = item.name
         batch_id = filename[: filename.rfind(".")]
-        n_data = 2046 if int(batch_id[-1]) == 0 else 440
+        n_data = 2046 if int(batch_id[-1]) == 0 else 441
```

Comment:
A dummy `PAGELOAD` origin has been added to count the number of sampled pageloads in a session.